src/main/java/ponyo/commands/ListCommand.java

```diff
@@ -0,0 +1,22 @@
+package ponyo.commands;
+
+import ponyo.data.task.TaskList;
+import ponyo.storage.Storage;
+import ponyo.ui.Ui;
+
+/**
+ * Displays the list of stored tasks.
+ */
+public class ListCommand extends Command {
```

Comment:
Same as the other classes that inherit from command maybe you could change the naming?